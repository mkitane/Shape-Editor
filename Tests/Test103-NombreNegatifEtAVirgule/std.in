C c1 1 -1 1
C c2 1 1.2 1.3
LIST
EXIT