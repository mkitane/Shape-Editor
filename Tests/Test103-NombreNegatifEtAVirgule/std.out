OK
ERR
C c1 1 -1 1